sshmuxer/requests.go

```diff
@@ -171,6 +177,15 @@ func handleRemoteForward(newRequest *ssh.Request, sshConn *utils.SSHConnection,
 			return
 		}
 
+		// reply that the session is successfully set up with the port that was
+		// assigned (which may be different than what was requested)
+		channelForwardReplyPayload := ssh.Marshal(channelForwardReply{bindPort})
```

Comment:
Just to confirm, does this work fine in situations with HTTP, TCP, and Alias forwards?

Reply:
Thinking about how to write a proper test, either with the compiled binary or preferably a smaller subset of the components in this library...

Reply:
Something like that would likely need to be tested with the compiled bin/with an SSH compliant client just to verify it does what is expected